FirstProject/new-evaluation.html

```diff
@@ -89,26 +87,35 @@ <h4 style="color:white;">Senior</h4>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">Should the candidate be hired?</h2>
-            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="The type of project that is suitable for the candidate
-Is guidance required for the candidate
-" ></textarea>
+            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="The type of project that is suitable for the candidate-->
+<!--Is guidance required for the candidate-->
+<!--" ></textarea>-->
+            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+
+            </div>
         </div>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">General Impression</h2>
-            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="*required
-" ></textarea>
+            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="*required-->
+<!--" ></textarea>-->
+            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+
+            </div>
         </div>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">Workflow, Leadership & Soft Skills</h2>
-            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="Describe the environment in which the candidate works.
-Describe any guidance or management experience
-" ></textarea>
+            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="Describe the environment in which the candidate works.-->
+<!--Describe any guidance or management experience-->
+<!--" ></textarea>-->
+            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+
+            </div>
         </div>
 
         <form class="language_form" style="text-align:center;">
-            <fieldset class="field_section" style="display:inline; width: 72%;">
+            <fieldset class="field_section" style="display:inline; width: 61%;">
```

Comment:
Now imagine how much easier it would've been to have this styling done against `.field_section`, for example. A single CSS change instead of a bunch of 'em, throughout the HTML. :)